roslibrust/src/lib.rs

```diff
@@ -105,6 +105,37 @@ pub use rosbridge::*;
 pub mod rosapi;
 
 #[cfg(feature = "ros1")]
-mod ros1;
-#[cfg(feature = "ros1")]
-pub use ros1::*;
```

Comment:
This had to be removed to make it possible to expose `Publisher`, `Subscriber`, etc. This will be a breaking change, but ROS1 support is marked as experimental. In the future, I'd like to remove the wildcard use statement for `rosbridge` as well, though this is a more impactful breaking change.